common-util/src/main/resources/externalresource.properties

```diff
@@ -95,3 +95,4 @@ sunbird_default_channel=
 cassandra_write_batch_size=100
 sunbird_telemetry_kafka_servers_config=localhost:9092,localhost:9093,localhost:9094
 sunbird_telemetry_kafka_topic=telemetryInsert
+sunbird_lms_telemetry_base_url=http://localhost:9000
```

Comment:
Rename 'sunbird_lms_telemetry_base_url'.  Let us follow this approach: sunbird_<service>_<var-name>.